scenarioo-client/app/pipes/metadata/metadataTreeListCreator.pipe.ts

```diff
@@ -12,21 +13,13 @@ export class MetadataTreeListCreatorPipe implements PipeTransform {
     constructor(private metadataTreeCreatorPipe: MetadataTreeCreatorPipe) {
     }
 
-    transform(metadata: any): any[] {
+    transform(metadata: any): any {
```

Comment:
it returns an array of object if I'm not wrong

Reply:
`transform(metadata: any): Array<object>`  is not working

Reply:
try `any[]` instead

Reply:
oh, I see, it is not returning an array, but really a generic object tree, then `any` is correct. Sorry.